contributions/catalog/webapps/templates/topnav.html

```diff
@@ -16,4 +17,4 @@
 
 <div class="dropdown-content" id="myDropdown" style="display: none">
     <a href="{{ url_for('contribute.logout') }}">Logout</a>
```

Comment:
In the home page (http://localhost:5050/catalog/), the logout button is not appearing. It could be a CSS thing.